doc/man7/EVP_MAC-KMAC.pod

```diff
@@ -47,8 +47,10 @@ The default value is 0.
 
 =back
 
-The "custom" and "key" parameters must be set before EVP_MAC_init().
+The "custom" parameter must be set before EVP_MAC_init().
```

Comment:
This line isnt quite right either. Since custom could now be set in the init..

Reply:
Good point again.